### PR TITLE
Decoded several IAnimationBuffer virtual functions

### PR DESCRIPTION
**Pull request by Mozz3d**

### CODE REVIEW

**Review by WopsS — DISMISSED** [review]
Thanks.

File: include/RED4ext/Scripting/Natives/animIAnimationBuffer.hpp [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+{
+namespace anim
+{
+struct IAnimationBuffer : ISerializable
+{
+    static constexpr const char* NAME = "animIAnimationBuffer";
+    static constexpr const char* ALIAS = NAME;
+
+    virtual bool Initialize() = 0;
+    virtual bool Load();
```
Comment (WopsS) [inline include/RED4ext/Scripting/Natives/animIAnimationBuffer.hpp:17]: I see the the rest of the class is virtual, do you need to define these methods then?

Comment (Mozz3d) [reply include/RED4ext/Scripting/Natives/animIAnimationBuffer.hpp:17]: Are you talking about defining the methods in general for IAnimationBuffer? Or the specific implementations?

If it's the latter case those are methods I found to have some level of base implementation, which I assumed could be relevant. But in hindsight it seems unnecessary

Comment (WopsS) [reply include/RED4ext/Scripting/Natives/animIAnimationBuffer.hpp:17]: The specific implementation for these methods https://github.com/WopsS/RED4ext.SDK/pull/157/files#diff-02e0f54aa59dc27be0144120e671cfde17e58c320b3d958b416466509d0605b9R7-R82.

Comment (WopsS) [reply include/RED4ext/Scripting/Natives/animIAnimationBuffer.hpp:17]: @Mozz3d can you remove these implementations?

**Pull request closed without merging**